src/components/Chat.js

```diff
@@ -72,10 +74,16 @@ export class Chat extends PureComponent {
      *  - `livestream dark`
      */
     theme: PropTypes.string,
+    /** The App locale */
+    locale: PropTypes.string,
+    /** The messages for the given locale */
+    messages: PropTypes.object,
   };
 
   static defaultProps = {
     theme: 'messaging light',
+    locale: 'en',
```

Comment:
@Isaakske do we still need this prop if we're not using it for setting moments locale

Reply:
also, can you give an example of how you set Moment.js locale so we can get the date's translated as well.

Reply:
[The documentation](https://github.com/formatjs/react-intl/blob/master/docs/Getting-Started.md#creating-an-i18n-context) shows the prop being passed, even when only the appropriate set of messages is passed. I don't know the exact reasoning behind it.

Reply:
A simple way to set it in the root component:

`import moment from 'moment';
import 'moment/locale/nl';

moment.locale('nl');`